### PR TITLE
Add Sentinel-1 indexing automation

### PR DESCRIPTION
**Pull request by alexgleith**
* Adds a DAG to index Sentinel-1 data in the Prod-AF Airflow

### CODE REVIEW

File: dags/sentinel-1_indexing.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+with dag:
+    INDEXING = KubernetesPodOperator(
+        namespace="processing",
+        image=INDEXER_IMAGE,
+        image_pull_policy="Always",
+        arguments=[
+            "sqs-to-dc",
+            "--stac",
+            "--update-if-exists",
+            "--allow-unsafe",
```
Comment (tebadi) [inline dags/sentinel-1_indexing.py:127]: Do you need this flag because of `update-if-exists`?

File: dags/sentinel-1_indexing.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    #     name="explorer-summary",
+    #     task_id="explorer-summary-task",
+    #     get_logs=True,
+    #     affinity=affinity,
+    #     is_delete_operator_pod=True,
+    # )
+
+    INDEXING
+    # INDEXING >> OWS_UPDATE_EXTENTS
+    # INDEXING >> EXPLORER_SUMMARY
```
Comment (tebadi) [inline dags/sentinel-1_indexing.py:173]: This commented block is placeholder for explore and ows so it's fine.

**Review by tebadi — APPROVED** [review]
The title is Sentinel-2, it should be Sentinel-1.

File: dags/sentinel-1_indexing.py (hunk 1 above)
Comment (alexgleith) [reply dags/sentinel-1_indexing.py:127]: Yep, that's right.

File: dags/sentinel-1_indexing.py (hunk 2 above)
Comment (alexgleith) [reply dags/sentinel-1_indexing.py:173]: 👍

**Pull request merged**